optimalPosition.py

```diff
@@ -0,0 +1,21 @@
+import csv
+
+#Given a string of current board config, function opens csv file, makes a dictionary, & gets value from dict with current board position
+def optimalMove(currentBoard):
```

Comment:
I think this can be improved by passing in a CSV file instead, in case we create multiple different files.